src/main/java/Entities/Board.java

```diff
@@ -34,8 +33,20 @@ public Player[] getPlayers(){
         return players;
     }
 
+    public void removePlayer(Player player){
+        List<Player> newPlayers = Arrays.asList(players);
```

Comment:
we might want to store Players as an ArrayList and refactor this implementation

Reply:
I agree, we should refactor into an arraylist to make it easier to support games with a variable number of players.

Reply:
I agree. If I'm understanding this correctly, your removePlayer method is creating a new copy of players each time a player is removed? An ArrayList is resizable and would contain a remove function, which I agree would be more suitable for storing players in a game like Monopoly.

Reply:
Yeah pretty difficult considering we might have to remove specific players in case of bankruptcy.